app/helpers/dashboard_helper.rb

```diff
@@ -0,0 +1,26 @@
+# frozen_string_literal: true
+
+module DashboardHelper
+  def dashboard_cards_by_role
+    {
+      challenge_manager: [
+        { image_path: 'folder_open', href: 'manage_submissions', alt: 'submissions', title: 'Submissions',
+          subtitle: 'Manage submissions, evaluations, and evaluators.' },
+        { image_path: 'content_copy', href: 'evaluation_forms', alt: 'evaluation forms', title: 'Evaluation Forms',
+          subtitle: 'Create and manage evaluation forms.' },
+        { image_path: 'map', href: 'user_guide', alt: 'user guides', title: 'User Guides',
+          subtitle: 'Learn how to make the most of the platform.' },
+        { image_path: 'support_agent', href: 'federal-agency-faqs', alt: 'help', title: 'Help',
+          subtitle: 'Get support on the Challenge.Gov platform.' }
+      ],
+      solver: [],
```

Comment:
we won't build a `solver` dashboard, at least for now, they will only access the logged-in features on the phoenix portal app.